cobigen-templates/templates-devon4j/src/main/templates/crud_java_server_app/templates/java/${variables.rootPackage}/${variables.component}/common/api/${variables.entityName}.java.ftl

```diff
@@ -2,9 +2,18 @@
 package ${variables.rootPackage}.${variables.component}.common.api;
 
 import ${variables.rootPackage}.general.common.api.ApplicationEntity;
+<#list pojo.fields as field>
+		<#if field.name="id">
```

Comment:
I don't think this is a good `if` condition for composite keys. I can have a field named as `id` but maybe I'm not using Composite keys. Please use the type of the field like `CompositeEmployeeKey` and you can see if it starts with the word `Composite`.

That seems to me a more valid version to check what the user wants.

Reply:
Comment incorporated. Re-review needed.